packages/app/src/styles/theme/_apply-colors.scss

```diff
@@ -283,6 +283,15 @@ ul.pagination {
       }
     }
   }
+
+  .grw-foldertree {
+    .list-group-item {
+      .grw-foldertree-title-anchor {
+        color: inherit;
+      }
+    }
```

Comment:
@mudana-grune 
Can it be commonized like,
```
  .grw-pagetree, .grw-foldertree {
    .list-group-item {
      .grw-pagetree-title-anchor {
        color: inherit;
      }
    }
  }
```

Reply:
This css part has been compacted